### PR TITLE
style(app): Update app styles

### PR DESCRIPTION
**Pull request by MarkPieszak**
Updated general app styles.
Obviously can spend more time & change/tweak anything you guys would like, just let me know!

@jeffwhelpley @gdi2290 

screen shot 2016-04-25 at 12 13 42 pm

### CODE REVIEW

**Comment by gdi2290** [comment]
+9001

File: src/index.html [hunk 1]
```diff
@@ -11,11 +11,15 @@
 
   <base href="/">
 
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="https://code.getmdl.io/1.1.3/material.indigo-pink.min.css">
+  <script defer src="https://code.getmdl.io/1.1.3/material.min.js"></script>
```
Comment (gdi2290) [inline src/index.html]: we have to move these script to the bottom

**Comment by gdi2290** [comment]
I like it but we should move the scripts to the bottom and async load the fonts

**Comment by MarkPieszak** [comment]
Ahh that's my fault I just took their CDN code from their website!

**Comment by gdi2290** [comment]
I'm not a fan of a js 3rd party module running outside of angular/build system. Is there any way to keep the css without the js? Maybe we can eventually have material2 provide a path to just give us the css

**Comment by MarkPieszak** [comment]
I'm actually not sure I'm using material light, I'll just remove it all together.

**Comment by gdi2290** [comment]
thanks!

**Pull request merged**